### PR TITLE
fixes #20031; uint64 is an ordinal type since 1.0

### LINKED ISSUE #20031: Cant use `uint64` in case
### Example
```nim
proc main(a: uint64) =
  case a
  else:
    discard

main(10)
```
https://play.nim-lang.org/#ix=44p1

### Current Output
```
Error: selector must be of an ordinal type, float or string
```

### Expected Output
```
```

```
$ nim -v
Nim Compiler Version 1.6.6 [Windows: amd64]
Compiled at 2022-05-05
Copyright (c) 2006-2021 by Andreas Rumpf

active boot switches: -d:release
```

### PR DESCRIPTION
**Pull request by ringabout**
fixes #20031

### CODE REVIEW

**Comment by Araq** [comment]
Seems to cause problem

**Comment by ringabout** [comment]
Sorry, the code had a typo.

File: compiler/semstmts.nim [hunk 1]
```diff
@@ -1050,7 +1050,7 @@ proc semCase(c: PContext, n: PNode; flags: TExprFlags): PNode =
   var typ = commonTypeBegin
   var hasElse = false
   let caseTyp = skipTypes(n[0].typ, abstractVar-{tyTypeDesc})
-  const shouldChckCovered = {tyInt..tyInt64, tyChar, tyEnum, tyUInt..tyUInt32, tyBool}
+  const shouldChckCovered = {tyInt..tyInt64, tyChar, tyEnum, tyUInt..tyUint64, tyBool}
```
Comment (ringabout) [inline compiler/semstmts.nim]:
```suggestion
  const shouldChckCovered = {tyInt..tyInt64, tyChar, tyEnum, tyUInt..tyUInt64, tyBool}
```

**Comment by metagn** [comment]
Backport?

**Comment by Araq** [comment]
Ping @narimiran please backport.

**Comment by narimiran** [comment]
> Ping @narimiran please backport.

Done.

**Pull request merged**